lua/test_assets/opponent_test_config.lua

```diff
@@ -89,6 +89,40 @@ return {
 		['template'] = '',
 		['type'] = 'solo',
 	},
+	exampleMatch2RecordSoloWithSpace = {
```

Comment:
maybe underscore instead of space?
and maybe add a testcase for one with actual space?